entities/weapons/ix_hands.lua

```diff
@@ -43,7 +43,7 @@ SWEP.ViewModel = Model("models/weapons/c_arms.mdl")
 SWEP.WorldModel = ""
 
 SWEP.UseHands = true
-SWEP.LowerAngles = Angle(0, 5, -14)
+SWEP.LowerAngles = Angle(0, 3, -6)
```

Comment:
Why was this changed?

Reply:
@ZeMysticalTaco because Angle(0, 5, -14) just hides ix_hands when they are lowered, wich is not proper considering altLower is disabled

Reply:
@ZeMysticalTaco because Angle(0, 5, -14) just hides ix_hands when they are lowered, wich is not proper considering altLower is disabled

Reply:
Okay.